product/dialogs/miq_dialogs/miq_provision_redhat_dialogs_template.yaml

```diff
@@ -439,7 +439,7 @@
               :dvs: true
               :vlans: true
             :method: :allowed_vlans
-          :description: Network
+          :description: Virtual Nic Profile id
```

Comment:
Should Nic be NIC since it is an acronym?

Reply:
Same with ID?

Reply:
done